### PR TITLE
EG-2854 Failing Test: net.corda.node.internal.NodeStartupCliTest.--nodeconf using absolute path will not be changed

### PR DESCRIPTION
**Pull request by pnemeth**
fix test failing on Windows

### CODE REVIEW

File: node/src/test/kotlin/net/corda/node/internal/NodeStartupCliTest.kt [hunk 1]
```diff
@@ -65,7 +67,7 @@ class NodeStartupCliTest {
     @Test(timeout=300_000)
     fun `--nodeconf using absolute path will not be changed`() {
         CommandLine.populateCommand(startup, CommonCliConstants.CONFIG_FILE, "/$customNodeConf")
```
Comment (JamesHR3) [inline node/src/test/kotlin/net/corda/node/internal/NodeStartupCliTest.kt]: Might be better off constructing this as a path and then calling `toString` on it.

**Review by JamesHR3 — CHANGES_REQUESTED** [review]
One small thing

**Review by JamesHR3 — APPROVED** [review]
LGTM

**Comment by fowlerrr** [comment]
Can I ask if this one can get merged?  It's probably just my personality type but all of those red `x`s in my PRs are driving me mad! 😄

**Pull request merged**